.gitattributes

```diff
@@ -24,3 +24,4 @@ utils/generator export-ignore
 utils/rules export-ignore
 typo3.constants.php export-ignore
 /stubs export-ignore
+/bin export-ignore
```

Comment:
are you sure this needs to be excluded?

Reply:
Yes, it is only for the generate command. I will show you. We had to create our custom console application.

Reply:
ah ok so only relevant when using the git source.